public/index.html

```diff
@@ -97,6 +97,10 @@
       style="--pwacompat-splash-font: 24px Virgil"
     />
 
+
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+
+
```

Comment:
also, what's this about? Did you add the reset to solve something? It's kinda breaking our UI in subtle ways :)

Reply:
it was just for a transition (animation effect) of a sidebar, replacing it with css.